API/package.json

```diff
@@ -10,7 +10,12 @@
   "license": "MIT",
   "dependencies": {
     "express": "^4.15.3",
+    "memory-cache": "^0.2.0",
+    "request": "^2.83.0",
     "socket.io": "^2.0.3",
     "web3": "^1.0.0-beta.22"
+  },
+  "devDependencies": {
+    "should": "^13.1.2"
```

Comment:
Can this be used with Polymer too? I think we should select a testing librabry that can be used front and back

Reply:
https://www.polymer-project.org/2.0/docs/tools/tests

Reply:
the ```should``` module is part of the mocha test-suite , which is part of web-component-tester. It's the de-facto way of unit testing nodejs code.